fiware_cloto/cloto/tests/test_auth_manager.py

```diff
@@ -61,24 +60,27 @@ def Session(self, auth, timeout):
 
 class AuthorizationManagerTests(TestCase):
     def setUp(self):
+        self.user = "user"
         self.token = "ff01eb8a8d69418c95f0009dda9bc1852"
+        self.token_not_expired = "aa01eb8a8d69418c95f0009dda9bc0000"
         self.token_other_tenant = "gg01eb8a8d69418c95f0009dda9bc1852"
         self.token_not_found = "0001eb8a8d69418c95f0009dda9bc1852"
         self.token_not_authorized = "bbbbbbba8d69418c95f0009dda9bc1852"
         self.fakeToken = "fffffffffffffffffffffffffffffffff"
         self.authToken = "77d254b3caba4fb29747958138136ffa"
         self.url = "http://130.206.80.61:35357/v2.0"
+        self.url_v3 = "http://130.206.80.61:35357/v3"
```

Comment:
See comment above: these should be "fake" URLs, as with the rest of attributes you define here.